docs/general/administration/hardware-acceleration/rockchip.md

```diff
@@ -229,6 +229,73 @@ Root permission is required.
+3. Verify OpenCL runtime status as following - example is collected from LXC runtime of Ubuntu Jammy - steps are the same as docker deployments:
+ - libmali user-space drivers should be installed inside LXC container, otherwise opencl=ocl@rk device won't be initialized
+
+```shell
+root@jellyfin:~# dpkg -l | egrep "libmali|clinfo|jellyfin"
+ii  clinfo                          3.0.21.02.21-1                          arm64        Query OpenCL system information
+ii  jellyfin                        10.9.11+ubu2204                         all          Jellyfin is the Free Software Media System.
+ii  jellyfin-ffmpeg6                6.0.1-8-jammy                           arm64        Tools for transcoding, streaming and playing of multimedia files
+ii  jellyfin-server                 10.10.0+ubu2204                         arm64        Jellyfin is the Free Software Media System.
+ii  jellyfin-web                    10.10.0+ubu2204                         all          Jellyfin is the Free Software Media System.
+ii  libmali-valhall-g610-g13p0-gbm  1.9-1                                   arm64        Mali GPU User-Space Binary Drivers
+
+root@jellyfin:~# /usr/lib/jellyfin-ffmpeg/ffmpeg -v debug -init_hw_device rkmpp=rk -init_hw_device opencl=ocl@rk
+ffmpeg version 6.0.1-Jellyfin Copyright (c) 2000-2023 the FFmpeg developers
+  built with gcc 11 (Ubuntu 11.4.0-1ubuntu1~22.04)
+  configuration: --prefix=/usr/lib/jellyfin-ffmpeg --target-os=linux --extra-version=Jellyfin --disable-doc --disable-ffplay --disable-ptx-compression --disable-static --disable-libxcb --disable-sdl2 --disable-xlib --enable-lto --enable-gpl --enable-version3 --enable-shared --enable-gmp --enable-gnutls --enable-chromaprint --enable-opencl --enable-libdrm --enable-libxml2 --enable-libass --enable-libfreetype --enable-libfribidi --enable-libfontconfig --enable-libbluray --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libopenmpt --enable-libdav1d --enable-libsvtav1 --enable-libwebp --enable-libvpx --enable-libx264 --enable-libx265 --enable-libzvbi --enable-libzimg --enable-libfdk-aac --arch=arm64 --cross-prefix=/usr/bin/aarch64-linux-gnu- --toolchain=hardened --enable-cross-compile --enable-rkmpp --enable-rkrga
+...
+
+arm_release_ver: g13p0-01eac0, rk_so_ver: 10
+[AVHWDeviceContext @ 0x55a5b30a00] 1 OpenCL platforms found.
+[AVHWDeviceContext @ 0x55a5b30a00] 1 OpenCL devices found on platform "ARM Platform".
+[AVHWDeviceContext @ 0x55a5b30a00] 0.0: ARM Platform / Mali-G610 r0p0
+[AVHWDeviceContext @ 0x55a5b30a00] cl_arm_import_memory found as platform extension.
+[AVHWDeviceContext @ 0x55a5b30a00] cl_khr_image2d_from_buffer found as platform extension.
+[AVHWDeviceContext @ 0x55a5b30a00] DRM to OpenCL mapping on ARM function found (clImportMemoryARM).
+Successfully parsed a group of options.
+...
+```
```

Comment:
please remove the prompts from the commands, and separate the commands themselves and the output of the commands into separate code blocks

Reply:
Also done. Please, refine if still required.